tests/unittests/PalTests.cpp

```diff
@@ -1,370 +1,16 @@
-#if 0
 //
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 //
 
 #include "common/Common.hpp"
-#include "common/MockISemanticContext.hpp"
+#include "pal/PseudoRandomGenerator.hpp"
+#include "Version.hpp"
 
 using namespace testing;
 
 class PalTests : public Test {};
 
-//---
```

Comment:
Why are all these tests being removed?

Reply:
These changes were never run. The whole file was commented out. I recovered some.

Reply:
In that case, there should be an issue tracking the tests not running. as it means there is test coverage gap right now.

Reply:
Testcases are commented out in lib\pal\PalTests.cpp #1010

Reply:
I have created a bug,